WordPress/src/main/java/org/wordpress/android/ui/whatsnew/FeatureAnnouncementDialogFragment.kt

```diff
@@ -40,14 +38,12 @@ class FeatureAnnouncementDialogFragment : DialogFragment() {
         viewModel = ViewModelProvider(this, viewModelFactory)
                 .get(FeatureAnnouncementViewModel::class.java)
 
-        if (VERSION.SDK_INT >= VERSION_CODES.M) {
-            val window: Window? = dialog.window
-            window?.let {
-                window.statusBarColor = dialog.context.getColorFromAttribute(attr.colorSurface)
-                if (!resources.configuration.isDarkTheme()) {
-                    window.decorView.systemUiVisibility = window.decorView
-                            .systemUiVisibility or View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
-                }
+        val window: Window? = dialog.window
+        window?.let {
+            window.statusBarColor = dialog.context.getColorFromAttribute(attr.colorSurface)
+            if (!resources.configuration.isDarkTheme()) {
+                window.decorView.systemUiVisibility = window.decorView
+                        .systemUiVisibility or View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
```

Comment:
Out of scope for this PR, but I noticed that this piece of code is exactly the same as the one in the fragment above, `StoriesIntroDialogFragment.kt`. Might be a good helper function to extract on the type of `dialog` or `Window`. 🤓

Reply:
My suggestion is to never go down that rabbit hole in WPAndroid and keep the changes relevant to the PR 😅